app/controllers/user/payments_controller.rb

```diff
@@ -9,13 +9,19 @@ class PaymentsController < ::ApplicationController
 
       def index
         begin
-          customer = DiscourseSubscriptions::Customer.find_by(user_id: current_user.id, product_id: nil)
+          customer = Customer.find_by(user_id: current_user.id)
+          products = Product.all.pluck(:external_id)
```

Comment:
minor thing, in all other places you are calling it `product_ids` and I think it is a better name than `products`